docs/prod-host-setup.md

```diff
@@ -216,6 +216,28 @@ containing other qdiscs, most classful qdiscs perform rate control.
   - `connlimit` - restricts the number of connections for a destination IP
     address/from a source IP address, as well as limit the bandwidth
 
+### Mitigating Block Device Issues
```

Comment:
Maybe `Limiting Block Device Throughput` or `Mitigating Noisy Neighbours`

Reply:
Sure, I can rephrase. How about `Mitigating Noisy-Neighbour Related Block Device Issues`?

Reply:
How does `Configuring block device resource access for a multi-tenant environment` sound?